.github/workflows/aws-vz-api-helper.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Determine work branch
-case "${CIRCLE_BRANCH}" in
```

Comment:
CircleCI gave us this for free, but it is now exported in the **Build** step of the new workflow in this PR.